### ISSUE TITLE
It is better to orgainze a main readme rather than use two seperate readme

### ISSUE THREAD

**Issue by HuaizhengZhang** [GuluDeemo/FogVideo#6]
1. You can keep the current two readme for details but the project should contain the main readme for an easy start.

2. Try to load lightweight mobilenet backbone Faster RCNN or SSD. The fog should load different cheap models with ease.

3. Please check the roadmap and label the completed task.

**Comment by GuluDeemo** [comment]
1. I have merged readme into one file, and add a report.md to show detailed experiment results.

2. I have trained a fasterrcnn-mobilenetv2, I put evaluation here for future reference:

IoU metric: bbox
 Average Precision  (AP) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.164
 Average Precision  (AP) @[ IoU=0.50      | area=   all | maxDets=100 ] = 0.304
 Average Precision  (AP) @[ IoU=0.75      | area=   all | maxDets=100 ] = 0.160
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.048
 Average Precision  (AP) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.188
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.248
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=  1 ] = 0.186
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets= 10 ] = 0.267
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.273
 Average Recall     (AR) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.078
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.306
 Average Recall     (AR) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.403

3. I am following the roadmap

**Comment by HuaizhengZhang** [comment]
Ok. It is better to use off-the-shelf models with a clear reference in our codes.

Now we can use this.

**Comment by GuluDeemo** [comment]
I use the mobilenetv2 backbone fasterrcnn described at the [pytorch torchvision repo](https://github.com/pytorch/vision/blob/c542137c612e28e098bc4d78258cb6438b3805f0/torchvision/models/detection/faster_rcnn.py#L100) and slightly change it from [num_classes of 2 to num_classes of 91](https://github.com/pytorch/vision/blob/c542137c612e28e098bc4d78258cb6438b3805f0/torchvision/models/detection/faster_rcnn.py#L135).  And the [training codes](https://github.com/pytorch/vision/blob/master/references/detection/README.md) I use are also from torchvision reference. Later I will put a detailed documentation under readme.
